### DISCUSSION TITLE
Allow incoming channels only by specific nodes (without restarting LND)

### DISCUSSION THREAD

**Discussion by microlancer** [lightningnetwork/lnd#8454] [Ideas]
Hello,

It would be useful for me to be able to allow incoming channels that are from specific nodes either by node ID or by giving the other node user a special key they can use when creating the channel. I don't know if that would require a protocol change.

Currently, I am doing this by temporarily decreasing my minimum channel size and then restoring it back to the maximum every time I want to accept a new channel, but this is very cumbersome and results in quite a long downtime.

Is there a way to do this without restarting?

**Comment by guggero** [comment]
https://docs.lightning.engineering/lightning-network-tools/lnd/channel-acceptor
